kedro-docker/features/steps/cli_steps.py

```diff
@@ -177,6 +177,22 @@ def exec_kedro_command(context, command):
         assert False
 
 
+@given("I have installed the project dependencies")
+def pip_install_dependencies(context):
+    """Install project dependencies using pip."""
+    reqs_path = "src/requirements.txt"
```

Comment:
I think this will fail on Windows cause the path separator is `\`, maybe we can use `pathlib` instead?

Reply:
But yeah let me update for good practice!

Reply:
Oh ignore me I misread win_unit_test build and thought we had windows tests! Which is actually not that relevant for kedro docker so it's fine.